package.json

```diff
@@ -5,8 +5,9 @@
   "scripts": {
     "lint:ts": "eslint src -c .eslintrc.json --ext ts",
     "test:unit": "mocha --require ts-node/register --require source-map-support/register --recursive test/**/**/*.test.ts test/**/*.test.ts test/*.test.ts",
+    "test:integration": "mocha -t 10000000 --require ts-node/register --require source-map-support/register --recursive integration_test/**/**/*.test.ts integration_test/**/*.test.ts integration_test/*.test.ts",
     "test:facilitator": "./test/facilitator.sh",
-    "test": "npm run test:unit && npm run test:facilitator",
+    "test": "npm run test:unit && npm run test:integration",
```

Comment:
please add a max timeout

Reply:
Max timeout is added in *test:integration*

Reply:
Already present in in *test:integration*